plugins/header.rst

```diff
@@ -0,0 +1,121 @@
+Header Plugin
```

Comment:
i think it makes sense to group those plugins in one doc page. but filename and title should be plural: header plugins